packages/react-compose/src/next/mergeProps.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { css } from '@uifabric/utilities';
+import { GenericDictionary } from './types';
+
+/**
+ * Helper which deep clones props, but respecting JSX and classnames.
+ * @param target
+ * @param propSets
```

Comment:
A brief description of the params here could be very helpful for people onboarding on this.

Reply:
done :)